### ISSUE TITLE
1.0 release

### ISSUE THREAD

**Issue by samualtnorman** [total-typescript/ts-reset#194]
I've used this package in production at work for 7 months now.
I know this package hasn't been updated in 8 months, but at the same time, I've not needed anything fixing or added to this package.
Don't you think this package is ready for a 1.0 release?

**Comment by mattpocock** [comment]
Yes, it probably is! The package feels very stable. I need to head down the list of issues and PR's to figure out if there's anything worth adding/changing.

**Comment by Sawtaytoes** [comment]
Do we still need it? I still have it, but I don't understand anymore.

I thought predicates 'n such were already fixed with `filter` in one of the prior releases (v5?). Am I missing something?
